src/system/applications/dialogs/roll-configuration.ts

```diff
@@ -29,6 +29,11 @@ export namespace RollConfigurationDialog {
          */
         parts: string[];
 
+        /**
+         * A dice formula stating any miscellanious other bonuses or negatives to the specific roll
```

Comment:
miscellaneous*

Reply:
Damn, thanks. Don't know why Code Spell Check didn't pick up on that. Annoying as it was late and I felt like the word wasn't right, but wasn't able to see why 🤣